.github/workflows/catnapw.yml

```diff
@@ -6,10 +6,6 @@ concurrency:
   group: catnapw
```

Comment:
Just a thought, should we create separate directories per OS instead of naming the files with a `w` prefix? What do you think?

One option is to defer this to a later point when we need more than 1 workflow files for Windows.

Reply:
The concurrent group is should be an unique identifier (it distinguishes which actions conflict), and we cannot use a directory based structure to achieve what you want.

Instead we can have suffixes like "catnap-linux" and "catnap-windows".

What do you think?

Reply:
Oh, I was referring to the filenames. e g., `.github/workflows/windows/catnap.yml`. 

Also, your suggestion about group names sounds good as well.

Reply:
Sure, I can rename them, but not sure that we can separate them into directories. See: https://stackoverflow.com/questions/64009546/how-to-run-multiple-github-actions-workflows-from-sub-directories